subprojects/docs/src/docs/release/notes.md

```diff
@@ -104,6 +104,21 @@ However, we strongly recommend that build and plugin authors use Gradle properti
 Memory usage for up-to-date checking has been improved.
 For the gradle/gradle build, heap usage dropped by 60 MB to 450 MB, that is a 12% reduction.
 
+### Build Scan Plugin default version updated to 1.16
+
+The default build scan plugin version has been updated to. This version provides you new insights of your build, for example:
+
+- Sources of slow configuration lifecycle callbacks
+- Build script compilation
+- Build deprecation warnings
+- Dependency repositories
+
+in build scans.
+
+You can enable build scans by running your build with `--scan`.
```

Comment:
:o: I don't think we need this in release notes.

Reply:
@melix after consulting with @tomdcc I've reworded the paragraph

Reply:
@melix and removed the empty promoted feature paragraph